File: Python/00-primero/00-script.py
#!/usr/bin/env python3

# Programa una funcion que cuente el numero de caracteres de una cadena de texto. 
# miFuncion("Hola mundo") devolvera 10

def contar_caracteres(cadena=""):
    if not cadena:
        print("No ingresaste ninguna cadena")
    else:
        print(f'La cadena "{cadena}" tiene {len(cadena)} caracteres')

# Ejemplos de uso
contar_caracteres()
contar_caracteres("Hola mundo")

File: Php/00-primero/00-script.php
<?php
/*
Programa una funcion que cuente el numero de caracteres de una cadena de texto. 
miFuncion("Hola mundo") devolvera 10 
*/

function contarCaracteres($cadena = "") {
    if (empty($cadena)) {
        echo "No ingresaste ninguna cadena\n";
    } else {
        echo "La cadena \"$cadena\" tiene " . strlen($cadena) . " caracteres\n";
    }
}

// Ejemplos de uso
contarCaracteres();           // Caso sin cadena
echo "\n";
contarCaracteres("Hola mundo"); // Caso con cadena
?>